src/transformers/highlight-directive.js

```diff
@@ -0,0 +1,53 @@
+// @ts-check
+const { highlightClassName } = require('../lineHighlighting');
+
+/**
+ * @param {object} attrs
+ * @param {string} className
+ */
+const addClassName = (attrs, className) => {
+  // should we add a check here if attrs.className exists?
+  attrs.className += ` ${className}`;
```

Comment:
`attrs.className` might not be defined yet. Let’s also return a new `attrs` object instead of mutating this one, just in case another transformer is holding a reference to it for some reason. (Mutating is probably slightly more efficient but it doesn’t really matter since this is happening at build time.)

Reply:
Done.